mne/viz/eyetracking/tests/test_heatmap.py

```diff
@@ -0,0 +1,42 @@
+# Authors: Scott Huberty <seh33@uw.edu>
+#
+# License: Simplified BSD
+
+import pytest
+
+import matplotlib.pyplot as plt
+import numpy as np
+
+import mne
+
+
+@pytest.mark.parametrize(
+    "axes",
+    [(None), (True)],
+)
```

Comment:
```suggestion
@pytest.mark.parametrize("axes", (None, True))
```
the inner parentheses are unneeded